pkg/webhooks/kafkatopic_validator.go

```diff
@@ -70,7 +79,7 @@ func (s *KafkaTopicValidator) validate(ctx context.Context, obj runtime.Object)
 		kafkaTopic.Name, fieldErrs)
 }
 
-func (s *KafkaTopicValidator) validateKafkaTopic(ctx context.Context, topic *banzaicloudv1alpha1.KafkaTopic, log logr.Logger) (field.ErrorList, error) {
+func (s *KafkaTopicValidator) validateKafkaTopic(ctx context.Context, log logr.Logger, topic *banzaicloudv1alpha1.KafkaTopic) (field.ErrorList, error) {
```

Comment:
One suggestion: it's probably better to get the logger from the context (follow what we have in the existing codebase) instead of passing it around as an argument, I believe passing the logger through argument is recommended

Reply:
I also think that passing the logger as argument is recommended and not to use ctx to get the logger.
Probably when you writing tests you can just use in the argument the log.Discard() to solve the dependency.

Reply:
> I also think that passing the logger as argument is recommended and not to use ctx to get the logger.

Oh really? Can you provide a reference? 

I thought it was the other way around since there was an onboarding task (https://github.com/banzaicloud/koperator/pull/765) that I did a long time ago to replace the logger argument with the one getting from the context. 

If it is the case that it's recommended to pass the logger as argument, we might need to revert that PR